pulp_container/app/urls.py

```diff
@@ -23,6 +24,7 @@
 router.register(r'^v2/(?P<path>.+)/blobs/uploads\/?', BlobUploads, basename='docker-upload')
 router.register(r'^v2/(?P<path>.+)/blobs', Blobs, basename='blobs')
 router.register(r'^v2/(?P<path>.+)/manifests', Manifests, basename='manifests')
+router.register(r'^v2/(?P<path>.+)/tags/list', TagsList, basename='tags-list')
```

Comment:
you should not register this viewset with the router. Add it as a view to the urlpatterns below. 

This router creates multiple routes for the viewset. We just need a single one.